.github/workflows/ci.yaml

```diff
@@ -59,9 +59,6 @@ jobs:
       - name: Display Go version
         run: go version
 
-      - name: Install dependencies
-        run: go get -t ./...
```

Comment:
A workspace side effect: adding a new package to rivershared seems to make it hard to resolve by `go get -t` despite `go test` and another commands working fine. No functional change by removing this since all Go commands will get dependencies, but another strike against workspaces, and a weird one.